MLS.Agent/DotnetKernelJupyterInstaller.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.CommandLine;
+using System.IO;
+using System.Threading.Tasks;
+using MLS.Agent.Tools;
+
+namespace MLS.Agent.Jupyter
+{
+    public static class DotnetKernelJupyterInstaller
+    {
+        public delegate Task<CommandLineResult> ExecuteCommand(string command, string args);
+
+        public static async Task<int> InstallKernel(ExecuteCommand executeCommand, IConsole console)
+        {
+            var dataPathsResult = JupyterPathInfo.GetDataPaths(await executeCommand("jupyter", "--paths"));
+            if (string.IsNullOrEmpty(dataPathsResult.Error))
+            {
+                Installkernel(dataPathsResult.Paths, console);
+                console.Out.WriteLine(".NET kernel installation succeded");
+                return 0;
+            }
+            else
+            {
+                console.Error.WriteLine($".NET Kernel Installation failed with error: {dataPathsResult.Error}");
+                return -1;
+            }
+        }
+
+        private static void Installkernel(IEnumerable<DirectoryInfo> dataDirectories, IConsole console)
+        {
+            foreach (var directory in dataDirectories)
+            {
+                if (directory.Exists)
+                {
+                    var kernelDirectory = directory.Subdirectory("kernels");
+                    if (!kernelDirectory.Exists)
+                    {
+                        kernelDirectory.Create();
+                    }
+
+                    var dotnetkernelDir = kernelDirectory.Subdirectory(".NET");
```

Comment:
path in kernel directory have problems with ```.``` in the name

Reply:
This results in a warning but it's not breaking anything. I think this is fine for now.